pandas/tests/reshape/test_pivot.py

```diff
@@ -2554,6 +2554,39 @@ def test_pivot_table_index_and_column_keys_with_nan(self, dropna):
 
         tm.assert_frame_equal(left=result, right=expected)
 
+    @pytest.mark.parametrize(
+        argnames=["index", "columns"],
+        argvalues=[("Category", "Value"), ("Value", "Category")],
```

Comment:
Just to agree with the style of the rest of the test-suite, can you do this instead:

```suggestion
        "index, columns",
        [("Category", "Value"), ("Value", "Category")],
```